builtInFunctions/multiESDTNFTTransfer.go

```diff
@@ -297,6 +297,10 @@ func (e *esdtNFTMultiTransfer) processESDTNFTMultiTransferOnSenderShard(
 			dstAddress,
 			listTransferData[i],
 			vmInput.ReturnCallAfterError)
+		if core.IsGetNodeFromDBError(err) {
+			log.Error("processESDTNFTMultiTransferOnSenderShard", "error", err, "token", string(listTransferData[i].ESDTTokenName))
```

Comment:
do we need this log here? We either return error or log it. Maybe a wrapping will help?

Reply:
Removed the log print. No wrapping needed.